src/test/Core.t.sol

```diff
@@ -208,28 +215,12 @@ contract CoreTest is CoreFixture {
 
         core.grantRole(C.WHITELISTED_ROLE, user);
         assertTrue(core.isWhitelisted(user));
+
+        vm.stopPrank();
```

Comment:
it wouldn't add much value in this case, but - would be nice if function modifiers could enforce post-requisites (is that a word?) and not just pre-requisite. I want for more python-decorator-esque behavior

Reply:
ooh yes we can do that
```
modifier prankAs(address user) {
  vm.startPrank(user);
  _;
  vm.stopPrank();
}
```

then we can just create a function like:
function xyz() prankAs(governor) { abc };